connectors/LeadDesk/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2193 @@
+                      "title": "Other information 24",
+                      "type": "string",
+                      "x-ms-visibility": "advanced"
+                    },
+                    "other25": {
+                      "description": "Other information 25",
+                      "title": "Other information 25",
+                      "type": "string",
+                      "x-ms-visibility": "advanced"
+                    },
+                    "other26": {
+                      "description": "Other information 26",
+                      "title": "Other information 26",
+                      "type": "string",
+                      "x-ms-visibility": "advanced"
+                    },
+                    "other27": {
+                      "description": "Other information 27",
+                      "title": "Other information 27",
+                      "type": "string",
+                      "x-ms-visibility": "advanced"
+                    },
+                    "other28": {
+                      "description": "Other information 28",
+                      "title": "Other information 28",
+                      "type": "string",
+                      "x-ms-visibility": "advanced"
+                    },
+                    "other29": {
+                      "description": "Other information 29",
+                      "title": "Other information 29",
+                      "type": "string",
+                      "x-ms-visibility": "advanced"
+                    },
+                    "other30": {
+                      "description": "Other information 30",
+                      "title": "Other information 30",
+                      "type": "string",
+                      "x-ms-visibility": "advanced"
+                    },
+                    "other31": {
+                      "description": "Other information 31",
+                      "title": "Other information 31",
+                      "type": "string",
+                      "x-ms-visibility": "advanced"
+                    },
+                    "other32": {
+                      "description": "Other information 32",
+                      "title": "Other information 32",
+                      "type": "string",
+                      "x-ms-visibility": "advanced"
+                    },
+                    "other33": {
+                      "description": "Other information 33",
+                      "title": "Other information 33",
+                      "type": "string",
+                      "x-ms-visibility": "advanced"
+                    },
+                    "other34": {
+                      "description": "Other information 34",
+                      "title": "Other information 34",
+                      "type": "string",
+                      "x-ms-visibility": "advanced"
+                    },
+                    "other35": {
+                      "description": "Other information 35",

```

Comment:
Why is there 35 'other information' parameter in this connector?

Reply:
Hi,
This is the way our db & system works.
We allow our clients (call center companies) to use these 35 customizable fields in case our basic info fields (e.g. name, birth date, etc) aren't enough. For instance if a company needs to input the caller's fidelity card number, or computer's model or whatnot, they can use some of these fields.
35 customizable options sounds like a lot but some companies might need them, and if they use our MS Flow connector to create or modify their contacts, they will need access to these fields.
We put them in the "advanced" section of the connectors so that the clients who don't use them don't need to scroll down all of them.
Hope this clarifies it.